### PR TITLE
Javadoc ndg

### PR DESCRIPTION
**Pull request by wwweert123**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -338,29 +338,75 @@ Example: `budget view`
...
 
-{Describe the value proposition: what problem does it solve?}
+Our financial planner application can help individuals manage their finances effectively and achieve their financial 
+goals. The purpose of such an application is to provide users with a range of tools and features to help them better 
+understand their financial situation. This will enable them to make more informed decisions, and plan for their future 
+financial well-being. The application will allow the user to keep track of their income, expenses and overall balance.
+It also lets user view their income and expenses using visualization tool to have a better view of their cash flow based
+on categories.  It also allows the user to set the budget for the month. It also allows users to add their financial
+goals to the wishlist. Furthermore, it allows users to track the stock market if they have interest in investing in 
+equities.
```
Comment (NeoMinWei) [inline docs/DeveloperGuide.md:360]: good details

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -338,29 +338,75 @@ Example: `budget view`
...
+| v1.0    | user                  | delete reminder               | delete reminders that I no longer want to keep track                                 |
+| v1.0    | user                  | mark reminder                 | set the reminder as completed                                                        |
+| v1.0    | user                  | view wishlist                 | keep track of my goals easily                                                        |
+| v1.0    | user                  | set goals                     | add a new goal to my that I think of                                                 |
+| v1.0    | user                  | delete goals                  | remove goals that I can no longer achieve                                            |
+| v1.0    | user                  | mark goal                     | that I have achieved                                                                 |
+| v1.0    | user                  | visualize my cash flow        | easily see where the distribution for my spending and earnings                       |
```
Comment (NeoMinWei) [inline docs/DeveloperGuide.md:396]: nice hard work

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -338,29 +338,75 @@ Example: `budget view`
...
+
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+Our financial planner application can help individuals manage their finances effectively and achieve their financial 
+goals. The purpose of such an application is to provide users with a range of tools and features to help them better 
+understand their financial situation. This will enable them to make more informed decisions, and plan for their future 
+financial well-being. The application will allow the user to keep track of their income, expenses and overall balance.
+It also lets user view their income and expenses using visualization tool to have a better view of their cash flow based
```
Comment (ryan1604) [inline docs/DeveloperGuide.md]: lets the user*

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -338,29 +338,75 @@ Example: `budget view`
...
 * Should work on main OS (Windows, Linux, Mac) that has Java 11 installed.
 * This app is meant for a single user.
 * This app is targeted towards users with an above-average typing speed.
+* Watchlist should work reliably and not crash the application when the 3rd party dependencies are down (API is down)
 
 ## Glossary
 
 * *Cashflow* - Refers to an income or expense.
 * *WishList* - A list containing goals/targets.
+* *Watchlist* - a list of stocks that the financial planner is currently tracking
```
Comment (ryan1604) [inline docs/DeveloperGuide.md]: Capitalise "a"

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -338,29 +338,75 @@ Example: `budget view`
...
 
 ## Instructions for manual testing
```
Comment (ryan1604) [inline docs/DeveloperGuide.md:412]: Want to add this too?

**Review by ryan1604 — APPROVED** [review]
LGTM

**Pull request merged**